lookout/core/slogging.py

```diff
@@ -110,10 +114,16 @@ def ensure_utf8_stream(stream):
 
     sys.stdout, sys.stderr = (ensure_utf8_stream(s)
                               for s in (sys.stdout, sys.stderr))
-    logging.basicConfig(level=level)
+
+    logging.basicConfig()
+    if config_path:
+        with open(config_path) as fh:
+            config = yaml.safe_load(fh)
+        for key, val in config.items():
+            logging.getLogger(key).setLevel(logging._nameToLevel.get(val, level))
     root = logging.getLogger()
-    # In some cases root has handlers and basicConfig() is a no-op
```

Comment:
This comment should be preserved, otherwise people will decide to optimize and remove "root.setLevel(level)" - I personally did that once to my own code and left that comment after.

Reply:
I removed the comment because before it was basicConfig(level=level) but now I call it without level and set level afterwards. It is now way less tempting to optimize but I can put it back anyway :)

Reply:
Good catch! We need to retain `basicConfig(level=level)` then because the root is not always set in the configuration file.

Reply:
we set it afterwards so it's ok. Isn't it?

Reply:
Yes indeed. We definitely need to comment on that :)